app/views/content/train-to-be-a-teacher/promos/_mailing-list-promo-salaries.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Advice and support", classes: %w[ml-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Find out more about teaching"
+  ) do %>
+      <p>Explore more benefits of teaching, including what it's really like to be in the classroom.</p>
```

Comment:
the first part and the second part of this sentence don't quite seem to go together - finding out what it's like to be in the classroom isn't a 'benefit of teaching'?